synapse/cortex.py

```diff
@@ -29,13 +29,57 @@ def __init__(self, name, msg):
         Exception.__init__(self, '%s invalid: %s' % (name, msg))
         self.param = name
 
+storectors = {
+    'lmdb': synapse.cores.lmdb.LmdbStorage,
+    'sqlite': synapse.cores.sqlite.SqliteStorage,
+    'ram': synapse.cores.ram.RamStorage,
+    'postgres': synapse.cores.postgres.PsqlStorage,
+}
+
 corctors = {
-    'lmdb': synapse.cores.lmdb.Cortex,
-    'sqlite': synapse.cores.sqlite.Cortex,
+    'lmdb': synapse.cores.lmdb.initLmdbCortex,
+    'sqlite': synapse.cores.sqlite.initSqliteCortex,
     'ram': synapse.cores.ram.initRamCortex,
-    'postgres': synapse.cores.postgres.Cortex,
+    'postgres': synapse.cores.postgres.initPsqlCortex,
 }
 
+def openstore(url, storconf=None, **opts):
```

Comment:
so to be clear - if someone wants the RAW storage backing, openstore() allows parsing and opening the store via url.  For example, opening the following:

```
store = synapse.cortex.openstore('sqlite:///./raw.db')
```
will MAKE the sqlite3 .db file, populate tables, and be done.  There will not be any rows represented nodes added to the store.